### PR TITLE
Update linter config to standard-with-typescript

### PR DESCRIPTION
**Pull request by sonallux**
See #195

### CODE REVIEW

**Comment by andreas-bauer** [comment]
Did you merged the recent changes into your branch?

**Comment by sonallux** [comment]
> Did you merged the recent changes into your branch?

Yes, I have rebased just now.

File: notification/src/api/rest/notificationConfigEndpoint.ts [hunk 1]
```diff
@@ -38,9 +36,9 @@ export class NotificationConfigEndpoint {
    */
   handleConfigSummaryRequest = async (req: express.Request, res: express.Response): Promise<void> => {
     const pipelineId = parseInt(req.params.id)
-    console.log(`Received request for configs with pipeline id ${pipelineId} from Host ${req.connection.remoteAddress}`)
+    console.log(`Received request for configs with pipeline id ${pipelineId}`)
```
Comment (georg-schwarz) [inline notification/src/api/rest/notificationConfigEndpoint.ts]: I think we should remove those logs, maybe replace them with logging middleware?

File: notification/src/api/rest/notificationConfigEndpoint.ts [hunk 2]
```diff
@@ -38,9 +36,9 @@ export class NotificationConfigEndpoint {
...
 
-    if (!pipelineId) {
+    if (isNaN(pipelineId)) {
       console.error('Request for config: ID not set')
```
Comment (georg-schwarz) [inline notification/src/api/rest/notificationConfigEndpoint.ts]: No need to log errors here (no server error, but client request error)

**Review by georg-schwarz — CHANGES_REQUESTED** [review]
Looks good to me.
Saw some logging issues (that don't originate from your code). Would you mind fixing them in your PR?

File: notification/src/api/rest/notificationConfigEndpoint.ts (hunk 1 above)
Comment (sonallux) [reply notification/src/api/rest/notificationConfigEndpoint.ts]: I am going to remove them. For a logging/tracing middleware we should open a separate issue.

**Comment by sonallux** [comment]
> Saw some logging issues (that don't originate from your code). Would you mind fixing them in your PR?

Done

**Review by georg-schwarz — APPROVED** [review]

**Pull request merged**